app/assets/javascripts/misc/pw-strength.js

```diff
@@ -35,7 +35,7 @@ function getFeedback(z) {
 
 function analyzePw() {
   const input = document.querySelector(
-    '#password_form_password, #update_user_password_form_password'
+    '#password_form_password, #reset_password_form_password, #update_user_password_form_password'
```

Comment:
rather than hardcode all these IDs, would it be easier to add a class to all of them?